db/migrate/20190801071143_drop_events_users_table.rb

```diff
@@ -0,0 +1,5 @@
+class DropEventsUsersTable < ActiveRecord::Migration[5.2]
+  def change
+    drop_table :events_users
```

Comment:
when dropping it is a good idea to replicate state so that reversing of migration is possible